gax/src/test/java/com/google/api/gax/rpc/RetryingTest.java

```diff
@@ -268,7 +274,12 @@ public void testUnknownStatusCode() {
             .build();
     UnaryCallable<Integer, Integer> callable =
         FakeCallableFactory.createUnaryCallable(callInt, callSettings, clientContext);
-    callable.call(1);
+    try {
+      callable.call(1);
+      Assert.fail("Callable should have thrown an exception");
+    } catch (Exception expected) {
```

Comment:
RuntimeException or something more specific. Other exceptions can still cause the test to fail.